internal/telemetry/tracker_test.go

```diff
@@ -0,0 +1,101 @@
+package telemetry
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"gopkg.in/segmentio/analytics-go.v3"
+)
+
+type testClient struct {
+	calls []interface{}
+}
+
+func (client *testClient) Enqueue(track analytics.Message) error {
+	client.calls = append(client.calls, track)
+	return nil
+}
+
+func (client *testClient) Close() error {
+	return nil
+}
+
+var testTime = time.Date(2021, 1, 2, 3, 4, 5, 6, time.UTC)
+
+func TestNoopTracker(t *testing.T) {
+	t.Run("should create a noop tracker", func(t *testing.T) {
+		tracker := noopTracker{}
+		assert.NotNil(t, tracker)
```

Comment:
this assert is not doing much here, we can probably get rid of this outer test-scope here